### ISSUE TITLE
TextEditor.onDidChangeScrollTop is deprecated.

### ISSUE THREAD

**Issue by rogeriopradoj** [georgebrindeiro/scroll-sync#23]
This is now a view method. Call TextEditorElement::onDidChangeScrollTop instead.
```
TextEditor.onDidChangeScrollTop (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:697:12)
<unknown> (/Users/rogerio/.atom/packages/scroll-sync/lib/scroll-sync.coffee:120:57)
ScrlSync.startTracking (/Users/rogerio/.atom/packages/scroll-sync/lib/scroll-sync.coffee:124:17)
atom-workspace.subToggle.add.atom.commands.add.scroll-sync:toggle (/Users/rogerio/.atom/packages/scroll-sync/lib/scroll-sync.coffee:31:28)
CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:259:29)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:59)
```

**Comment by trunghieu7393** [comment]
same issue.
i hope author fix soon :)

**Comment by Joao-Neves** [comment]
Same issue!

**Comment by FabianMeul** [comment]
Same issue :-)

**Comment by mark-hahn** [comment]
I am really surprised at how many people are using this.  Someone needs to
take ownership.

**Comment by georgebrindeiro-vtex** [comment]
@rogeriopradoj @trunghieu7393 @Joao-Neves @FabianMeul I just took ownership and released version 0.2.3, check it out and see if it works now!
